src/workerd/api/r2-api.capnp

```diff
@@ -18,15 +18,17 @@ struct R2BindingRequest {
     head @1 :R2HeadRequest $Json.flatten();
     get @2 :R2GetRequest $Json.flatten();
     put @3 :R2PutRequest $Json.flatten();
-    list @4 :R2ListRequest $Json.flatten();
-    delete @5 :R2DeleteRequest $Json.flatten();
-    createBucket @6 :R2CreateBucketRequest $Json.flatten();
-    listBucket @7 :R2ListBucketRequest $Json.flatten();
-    deleteBucket @8 :R2DeleteBucketRequest $Json.flatten();
-    createMultipartUpload @9 :R2CreateMultipartUploadRequest $Json.flatten();
-    uploadPart @10 :R2UploadPartRequest $Json.flatten();
-    completeMultipartUpload @11 :R2CompleteMultipartUploadRequest $Json.flatten();
-    abortMultipartUpload @12 :R2AbortMultipartUploadRequest $Json.flatten();
+    copy @4 :R2CopyRequest $Json.flatten();
```

Comment:
Need to check with @kentonv on this... typically changing the ordinal of a field is breaking. Is this ok or should `copy` be moved to the end?